rust/src/ssh/logger.rs

```diff
@@ -28,11 +28,15 @@ fn log_ssh(tx: &SSHTransaction, js: &mut JsonBuilder) -> Result<bool, JsonError>
         if tx.cli_hdr.swver.len() > 0 {
             js.set_string_from_bytes("software_version", &tx.cli_hdr.swver)?;
         }
-        if tx.cli_hdr.hassh.len() > 0 {
-            js.set_string_from_bytes("hassh", &tx.cli_hdr.hassh)?;
-        }
-        if tx.cli_hdr.hassh_string.len() > 0 {
-            js.set_string_from_bytes("hassh.string", &tx.cli_hdr.hassh_string)?;
+        if tx.cli_hdr.hassh.len() > 0 || tx.cli_hdr.hassh_string.len() > 0 {
+            js.open_object("hassh")?;
```

Comment:
Avoid type colision by adding this `hassh` object with `hash` and `string` members like with `ja3` and `ja3s`.

Reply:
What do you mean ?
I do not see the bug...